packages/lib/modules/swap/useIsPoolSwap.tsx

```diff
@@ -0,0 +1,6 @@
+import { usePathname } from 'next/navigation'
+
+export function useIsPoolSwap() {
+  const pathname = usePathname()
+  return pathname.includes('/pools') && pathname.includes('/swap')
+}
```

Comment:
What happens if we allow pool swaps directly in the swap UI eventually?

Reply:
Refactored into `isPoolSwap` + `useIsPoolSwapUrl`